weapon_list.py

```diff
@@ -25,6 +26,8 @@ def _sort_weapons(self):
         for weapon in self.weapon_list:
             if (weapon.weapon_type == WeaponType.normal):
                 self.normal_list.append(weapon)
+            if (weapon.weapon_type == WeaponType.normal2):
+                self.normal2_list.append(weapon)
```

Comment:
There is no need to have a normal2 list, just put them in the normal list. Can do regex of 'normal' in weapon.weapon_type.name or 'normal' in weapon.weapon_type.name. This applies to all other places that normal2 is duplicated from normal.